tests/Browser/Pages/HomePage.php

```diff
@@ -41,7 +41,7 @@ public function elements()
         return [
             '@form' => '#items-form',
             '@items-input' => '@form textarea',
-            '@not-found' => 'p.not-found'
+            '@not-found' => 'div.not-found'
```

Comment:
that line is actually obsolete, isn't it? But I think it's unrelated to this task.

Reply:
Yes, this is in main. It was introduced in a PR that has already been merged. I can remove it in this task. Should i?

Reply:
Sure, why not. Actually, it is in fact related to this task, right? (even though it's the `HomePage` test)

Reply:
Done.